efaar_benchmarking/efaar.py

```diff
@@ -74,14 +77,14 @@ def centerscale(features: np.ndarray, metadata: pd.DataFrame = None, plate_col:
     return features
 
 
-def embed_align_by_pca(
+def embed_by_pca(
```

Comment:
maybe `embed_by_pcaw` to emphasize the whitening step

Reply:
I actually removed the whitening step from this function since we do it as an additional step in the notebooks.

Reply:
Isn't this step still whitening https://github.com/recursionpharma/EFAAR_benchmarking/pull/33/files/7b92f4d14eba6638935575562c3c5d46e78930b6#diff-643359a6a9b4d94cee8d9b8e4e2fb113bf9243171e734a8d91700b03a0d56ea5R106?